### PR TITLE
testing new doc finder

### PR DESCRIPTION
**Pull request by derrickdso**

### CODE REVIEW

File: wp/wp-content/themes/phila.gov-theme/partials/departments/v2/document-finder.php [hunk 1]
```diff
@@ -8,8 +8,8 @@
 if ($vue_or_not) {
   $app_id = 'vue-app';
 
-  $vuejs_js_ids = ['https://philagov-vue-apps.s3.amazonaws.com/document-finder/v2/js/chunk-vendors.js', 'https://philagov-vue-apps.s3.amazonaws.com/document-finder/v2/js/app.js'];
-  $vuejs_css_ids = ['https://philagov-vue-apps.s3.amazonaws.com/document-finder/v2/css/chunk-vendors.css', 'https://philagov-vue-apps.s3.amazonaws.com/document-finder/v2/css/app.css'];
+  $vuejs_js_ids = ['https://philagov-vue-apps.s3.amazonaws.com/document-finder/dev/js/chunk-vendors.js', 'https://philagov-vue-apps.s3.amazonaws.com/document-finder/dev/js/app.js'];
```
Comment (karissademi) [inline wp/wp-content/themes/phila.gov-theme/partials/departments/v2/document-finder.php:11]: Did you mean to leave the dev links this PR?

File: wp/wp-includes/media.php [hunk 2]
```diff
@@ -3473,6 +3473,7 @@ function wp_prepare_attachment_for_js( $attachment ) {
 		'editLink'      => false,
 		'meta'          => false,
 		'mediaCategory' => $attachment_term,
+		'label'					=> $attachment->phila_label,
```
Comment (karissademi) [inline wp/wp-includes/media.php:3476]: What's the purpose of this change? It will get overridden on a WordPress update.

File: wp/wp-content/themes/phila.gov-theme/partials/departments/v2/document-finder.php (hunk 1 above)
Comment (derrickdso) [reply wp/wp-content/themes/phila.gov-theme/partials/departments/v2/document-finder.php:11]: Thats where the most up-to-date version of the doc finder app is (as opposed to v2) but I should duplicate the dev stuff into a prod folder and point these to prod

File: wp/wp-includes/media.php (hunk 2 above)
Comment (derrickdso) [reply wp/wp-includes/media.php:3476]: That's the label field added to the media items for the doc finder to use. I guess we will need to flag these before wp updates, because mediaCategory would also get overridden

File: wp/wp-content/themes/phila.gov-theme/partials/departments/v2/document-finder.php (hunk 1 above)
Comment (karissademi) [reply wp/wp-content/themes/phila.gov-theme/partials/departments/v2/document-finder.php]: Yes, let's make sure we're always pointing to the "prod" version of these things. Too confusing to have production in one place pointing to dev in another haha
